src/pages/login/login.page.jsx

```diff
@@ -138,10 +142,19 @@ class Login extends Component {
                     {this.state.errors.password.length > 0 ? <span className="errors">{this.state.errors.password}</span> : null}
 
                     <Button buttonType="primary" handleClick={this.handleSubmit}>Submit</Button>
+                    {this.props.isLoggedIn &&
+                        <Button buttonType="danger" handleClick={() => this.props.LogOut()}>Logout</Button>}
                 </div>
             </Frame>
         );
     }
 }
-
-export default Login;
\ No newline at end of file
+const mapStateToProps = (state) => ({
+    errorMessage: state.auth.errorMessage || state.user.errorMessage, //TODO: check if this is fine
```

Comment:
This is fine or not btw?

Reply:
abtak to fine. both messages are mutually exclusive and if 1 is error, 2nd will have to be null either because it isnt called or because it gave a success for the next call to actually execute and fail